specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -673,7 +673,7 @@
         },
         "deletedTime": {
           "type": "string",
-          "format": "date-time",
+          "format": "date-time-rfc1123",
```

Comment:
Will this break existing SDK's and usage by the customers?

Reply:
As the .net SDK test pass, should be good to .net SDK.
But I see Go SDK test fail, The Go SDK owner might need to look if it caused go SDK breaking.